modal/app.py

```diff
@@ -656,6 +656,7 @@ def function(
         # With `max_inputs = 1`, containers will be single-use.
         max_inputs: Optional[int] = None,
         i6pn: Optional[bool] = None,  # Whether to enable IPv6 container networking within the region.
+        untrusted: bool = False,  # Whether the function is untrusted (e.g. user-provided code)
```

Comment:
Actually (sorry for multiple rounds of review) looking at the parameter list it might make sense to put this next to `block_network` since they're sort of similar functionality.

Which also makes me wonder if there's a more general way we could expose restrictions so we don't end up with lots of individual piecemeal parameters in this overly-long function signature 🤔.